Langevin Test Fails on the institute machines

```
./pypresso ../testsuite/python/langevin_thermostat.py 
...F../testsuite/python/langevin_thermostat.py:49: DeprecationWarning: Passing normed=False is deprecated, and has no effect. Consider passing the density argument instead.
  vel[:, i], range=(-minmax, minmax), bins=n_bins, normed=False)
.s..

FAIL: test_03__friction_rot (__main__.LangevinThermostat)
Tests the rotational friction-only part of the thermostat.

Traceback (most recent call last):
  File "../testsuite/python/langevin_thermostat.py", line 184, in test_03__friction_rot
    o0 * np.exp(-gamma_r_i / rinertia * system.time), atol=5E-4)
  File "/tikhome/jlandsgesell/.local/lib/python3.6/site-packages/numpy/testing/_private/utils.py", line 1501, in assert_allclose
    verbose=verbose, header=header, equal_nan=equal_nan)
  File "/tikhome/jlandsgesell/.local/lib/python3.6/site-packages/numpy/testing/_private/utils.py", line 827, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0.0005

Mismatch: 100%
Max absolute difference: 0.00050059
Max relative difference: 0.00014899
 x: array([3.359489, 3.359489, 3.359489])
 y: array([3.35999, 3.35999, 3.35999])

----------------------------------------------------------------------
Ran 8 tests in 0.506s

FAILED (failures=1, skipped=1)
```

--- a/testsuite/python/langevin_thermostat.py
+++ b/testsuite/python/langevin_thermostat.py
@@ -155,7 +155,7 @@ class LangevinThermostat(ut.TestCase):
         gamma_r_a = np.array((1.5, 0.7, 1.2))
         o0 = np.array((5., 5., 5.))
 
-        system.time_step = 0.0005
+        system.time_step = 0.0001
         system.part.clear()
         system.part.add(pos=(0, 0, 0), omega_body=o0, rotation=(1, 1, 1))
         if espressomd.has_features("ROTATIONAL_INERTIA"):

